lib/renuo/bin-check/default-scripts/default_rails.rb

```diff
@@ -0,0 +1,160 @@
+module RenuoBinCheck
+  # rubocop:disable Metrics/ModuleLength
+  module DefaultScripts
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
+    # :reek:TooManyStatements
+    def rails
```

Comment:
To avoid method length they could be in seperated files in a folder. But that would be to expensive to call every time.